scalameta/trees/shared/src/main/scala/scala/meta/internal/prettyprinters/TreeSyntax.scala

```diff
@@ -149,7 +149,6 @@ object TreeSyntax {
         templ.self.name.is[Name.Anonymous] &&
         templ.self.decltpe.isEmpty &&
         templ.stats.isEmpty) s()
-      else if (templ.inits.nonEmpty || templ.early.nonEmpty) s(" extends ", templ)
```

Comment:
i don't understand how this is helping. this method (`templ`) is called on line 925 for `Defn.Class`, 936 for `Defn.Trait`, 971 for `Defn.Enum`, 993 for `Defn.Object` and 1016 for `Pkg.Object`. so for those cases it should work.

below, you are covering 4 out of these 5 cases (no `Pkg.Object`). perhaps the original submitter meant some other cases which we aren't covering yet. such as `Defn.Given`? should we ask for examples?

also, I would add plenty of tests, to make sure something didn't work before but is working now.

Reply:
understood. template needs to print extends, not the outer tree.

Reply:
Added some tests that should additionally highlight the issue

Reply:
> `Defn.Given` cannot have `extends` and I forgot about Pkg.Object.

didn't see this, sorry. still, would be good to have tests for all constructs which use template.

Reply:
so, should this method be removed completely? it doesn't do anything anymore.